scripts/calculator.js

```diff
@@ -61,10 +80,11 @@ const clickListener9 = () => {
 const clickListener0 = () => {
   addNumber('0')
 }
-
-/* Update this so that it also empties state. */
 const clickListenerClear = () => {
   updateScreen()
+  for (let firstNumber in state) {
```

Comment:
We should always try to name things what they are.

In this case, the `let` you're declaring here isn't actually "firstNumber". It's a key that'll be a different string each time the loop runs. (once it'll be 'firstNumber', then it'll be 'operator', then 'secondNumber')

If I were writing this, i'd do something along the lines of:
`for (let key in state) {`